onyo/commands/new.py

```diff
@@ -5,13 +5,50 @@
 
 from onyo import OnyoRepo
 from onyo.lib.commands import fsck, new as new_cmd
+from onyo.shared_arguments import template, path
 
 if TYPE_CHECKING:
     import argparse
 
 logging.basicConfig()
 log: logging.Logger = logging.getLogger('onyo')
 
+arg_template = dict(
+    metavar='TEMPLATE',
+    required=False,
+    type=template,
+    help='Specify the template to seed the new asset(s)')
+
+arg_edit = dict(
+    required=False,
+    default=False,
+    action='store_true',
+    help='Open new assets to edit them before creation')
+
+arg_keys = dict(
+    required=False,
+    metavar="KEYS",
+    nargs='+',
+    help=(
+        'Set key-value pairs in the new asset(s). Multiple pairs can be '
+        'specified (e.g. key=value key2=value2)'))
+
+arg_path = dict(
+    metavar='ASSET',
+    type=path,
+    nargs='*',
+    help=(
+        'Specify the directory and name of the new asset(s) '
+        '(in the format DIR/ASSET). Excludes usage of --tsv'))
+
+arg_tsv = dict(
+    metavar='TSV',
+    required=False,
+    type=path,
+    help=(
+        'Read information of new assets from a tsv file describing them. '
+        'Excludes the usage of --path'))
```

Comment:
I applied the Change in text mentioned above.

I created an issue for explaining the format in the help text (#395), since I think the explanation of the format should be comprehensible, and I am currently struggling with a good description.